src/Sarif/Core/Run.cs

```diff
@@ -220,6 +220,12 @@ public bool ShouldSerializeGraphs()
             return this.Graphs.HasAtLeastOneNonDefaultValue(Graph.ValueComparer);
         }
 
+        public bool ShouldSerializeAutomationDetails()
```

Comment:
>ShouldSerializeAutomationDetails [](http://example.com/codeflow?start=20&length=32)

Why did you only test for these two properties? AutomationDetails has many more.

Reply:
Adding the remaining properties.
thanks!